src/main/java/io/jenkins/plugins/managepermission/ManagePermissionEnabler.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.plugins.managepermission;
+
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import hudson.Extension;
+import hudson.init.InitMilestone;
+import hudson.init.Initializer;
+import jenkins.model.Jenkins;
+
+@Extension
+@Restricted(NoExternalUse.class)
+public class ManagePermissionEnabler {
+
+    @Initializer(after = InitMilestone.PLUGINS_STARTED)
+    public static void enableManagePermission() {
+        if (System.getProperty("jenkins.security.ManagePermission") == null) {
```

Comment:
Need to check if it's `true` instead of `null`

Reply:
With this, the deafult value is false, unless the property is exactly "true"

```suggestion
Booleans.getBoolean("jenkins.security.ManagePermission")
```

Reply:
@EstherAF no it's not working, because when property is not set it's the same as false, and we want to enable it in that case (without needed a System property).

Reply:
I think we want the permission to be enabled, even if the System property is not set. So testing the null is fine, because if the property is set => take property value (true/false) if no property set enable it.

Reply:
Sorry, my first comment in the other PR was misleading. The null check is fine, it means _enable the permission unless `jenkins.security.ManagePermission=false`_ (which is quite weird BTW, I don't think anyone would set the that to false).

Reply:
I it's possible, someone will do it one day, this is my mantra :)